src/components/common/PriceCalculation.vue

```diff
@@ -3,13 +3,38 @@
     <div class="row">
       <div class="col-sm-12">
           <div class="row">
+            <div v-if="cartLike.discountCodes && cartLike.discountCodes.length"
```

Comment:
I think we need to extract this part into a component, but that means too much sending events around. Leave this comment open, I need to think about it.